engine/class_modules/sc_death_knight.cpp

```diff
@@ -8934,6 +8974,11 @@ double death_knight_t::composite_leech() const
     m += buffs.voracious -> data().effectN( 1 ).percent();
   }
 
+  if ( dbc -> ptr && legendary.vampiric_aura.ok() && buffs.vampiric_blood -> up() )
```

Comment:
```suggestion
  if ( dbc -> ptr && legendary.vampiric_aura.ok() && buffs.vampiric_blood -> check() )
```

Reply:
Why would you remove uptime tracking here?

Reply:
Because it is in a composite / const function which can be called an arbitrary amount of times.